src/approximations/cvi.jl

```diff
@@ -58,6 +69,20 @@ const CVI = CVIApproximation
 # CVI implementations
 #---------------------------
 
+struct ForwardDiffGrad end
+
+function compute_grad(::ForwardDiffGrad, A::F, vec_params) where {F}
```

Comment:
How do these functions work in the combination with `get_df_m` and others?

Reply:
We do not have the `get_df_m`-like functions now.